docs/diagrams/UiClassDiagram.puml

```diff
@@ -4,57 +4,61 @@ skinparam arrowThickness 1.1
 skinparam arrowColor UI_COLOR_T4
 skinparam classBackgroundColor UI_COLOR
 
-package UI <<Rectangle>>{
-Class "<<interface>>\nUi" as Ui
-Class "{abstract}\nUiPart" as UiPart
-Class UiManager
-Class MainWindow
-Class HelpWindow
-Class ResultDisplay
-Class PersonListPanel
-Class PersonCard
-Class StatusBarFooter
-Class CommandBox
+package UI <<Rectangle>> {
+    Class "<<interface>>\nUi" as Ui
+    Class "{abstract}\nUiPart<T>" as UiPart
+    Class UiManager
+    Class MainWindow
+    Class HelpWindow
+    Class ResultDisplay
+    Class StudentListPanel
+    Class StudentCard
+    Class StatusBarFooter
+    Class CommandBox
+    Class StudentProfile
```

Comment:
I like how you add the new class for better readability and visualization of UI element